frontend/src/metabase/nav/components/DatabasePromptBanner/DatabasePromptBanner.styled.ts

```diff
@@ -0,0 +1,60 @@
+import styled from "@emotion/styled";
+import Button from "metabase/core/components/Button/Button";
+import ExternalLink from "metabase/core/components/ExternalLink";
+
+import { color } from "metabase/lib/colors";
+import { breakpointMinSmall } from "metabase/styled-components/theme";
+
+// This color is only used here, so I'm reluctant to add it to the color palette.
+const VIBRANT_BLUE = "#1888EC";
+
+export const DatabasePromptBannerRoot = styled.div`
+  background: ${VIBRANT_BLUE};
+  color: ${color("white")};
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+
+  ${breakpointMinSmall} {
```

Comment:
Add breakpoints to various elements to make the mobile view looks more polished. [The design is here](https://www.figma.com/file/H8NL31kXy2bNxuXtSKX0Fu/Really-encourage-admins-to-add-a-DB?type=design&node-id=27-92&t=ktyrZcQNXHgbNUrR-0).

#### Before
- ![image](https://github.com/metabase/metabase/assets/1937582/ebf5f156-088b-4e88-aefa-c2d0bb1c6145)
- ![image](https://github.com/metabase/metabase/assets/1937582/10ae7ab5-231a-459a-8019-fe54fde12790)

#### After
- ![image](https://github.com/metabase/metabase/assets/1937582/aa56a297-aee7-4159-98f8-990b5896a77f)
- ![image](https://github.com/metabase/metabase/assets/1937582/ce2adf87-676f-4b0a-ba93-4e0b5ed36de7)